packages/packagers/ts/src/TSPackager.js

```diff
@@ -0,0 +1,23 @@
+// @flow strict-local
```

Comment:
Is this only for packaging type definitions? Maybe the name of the file could include "types"?